src/components/Layout/LayoutView.js

```diff
@@ -79,8 +81,6 @@ const Layout = ({ classes, isSidebarOpened }) => (
             <PrivateRoute path="/adminportal" access={['admin']} component={fakeAdminView} />
             <PrivateRoute path="/admin/edit/:id" access={['admin']} component={editUserController} />
             <PrivateRoute path="/admin/view/:id" access={['admin']} component={viewUserController} />
-            <PrivateRoute path="/admin" access={['admin']} component={adminController} />
```

Comment:
@ogwuazors This was added in separate PR. It's overwritten section.  It's removing new features.